changelog.txt

```diff
@@ -1,8 +1,18 @@
 Version 3.3.0, April 1, 2020
 
+Renamings:
+  "unchecked code defaults" to "conservative defaults".
```

Comment:
The concept that used to be called "unchecked code defaults" is now called "conservative defaults".  This is a documentation change only.  I could remove this line if you think it is confusing.

Reply:
I agree with the general change we decided on on Monday. However, just calling it "conservative defaults" is confusing, because these defaults don't apply to checked code.
In our discussions on Monday we had settled on keeping the "unchecked" part.
So the documentation should talk about "conservative defaults for unchecked code" or "conservative unchecked code defaults" to distinguish them from the checked code defaults.

Reply:
I was trying to distinguish between:
 * the concept of conservative defaults
 * where they are applied.

I am going to just remove this line from the changelog, since it causes more harm than good.  Thanks for the feedback about that.

I think that text and naming elsewhere is probably OK, though changes are coming.  (This pull request cleared the way for more focused pull requests that will be easier to discuss on their own.)